.github/workflows/release.yml

```
@@ -27,13 +27,13 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: 14
+          node-version: 20
```

22 is the latest LTS version, we're already using that for `file-selector`, so we might as well here too.
```suggestion
          node-version: 22
```